src/cascadia/TerminalApp/TerminalPage.cpp

```diff
@@ -0,0 +1,28 @@
+﻿#include "pch.h"
+#include "TerminalPage.h"
+#if __has_include("TerminalPage.g.cpp")
+#include "TerminalPage.g.cpp"
+#endif
+
+using namespace winrt;
+using namespace Windows::UI::Xaml;
+
+namespace winrt::TerminalApp::implementation
+{
+    TerminalPage::TerminalPage()
+    {
+        // Xaml will by default attempt to load from ms-appx://TerminalApp/TerminalPage.xaml.
+        // We'll force it to load from the root of the appx instead.
+        const winrt::Windows::Foundation::Uri resourceLocator{ L"ms-appx:///TerminalPage.xaml" };
```

Comment:
Is this going to blow up with the other appx work you're working on?

Reply:
Yep, but I'll fix this when I fix that.